docs/guides/earning-cred.md

```diff
@@ -1,42 +1,41 @@
 ---
 title: 💧 Earning Cred
-description: How to get Cred from your contributions.
+description: How you get Cred from your contributions.
 ---
 
-As soon as you interact with SourceCred, you will be part of the Contribution Graph (i.e. a network of connected contributions). Being part of this Graph is like having your own pond now that holds [Cred]. Cred will flow in&out in different ways, just like water in a pond. 
+As soon as you interact with SourceCred, you will be part of the Contribution Graph (i.e. a network of connected contributions). Being part of this graph is like having your own pond that holds [Cred]. Cred will flow in and out in different ways, just like water in a pond. 
```

Comment:
We introduce the concept of a pond before we introduce the concept of water. It works, but could flow better. Suggest something along the lines of:

> As soon as you interact with SourceCred, you will be part of the Contribution Graph (i.e. a network of connected contributions). You and each contribution you make will be represented as a separate node in the graph, each with a Cred score. How Cred flows between nodes is dependent on the data source, configuration weights and other factors. But at a high level, as a metaphor, you can visualize Cred flowing through the graph as water flowing through an ecosystem of ponds and creeks.

Reply:
Thanks! Using this